lib/vehicle.rb

```diff
@@ -5,14 +5,18 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
+              :engine  
+  
+  attr_accessor :registration_date, :plate_type
 
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @plate_type = vehicle_details[:plate_type]
```

Comment:
I wouldn't put this as a static instance variable inside here. Instead make it its own method that is responsible for setting the plate type based on its age (you can still use the `antique?` and `electric_vehicle?` methods inside.